populateDB/migrations/005_add-seeding-helpers.sql

```diff
@@ -0,0 +1,36 @@
+--- Create function to handle adding a legacy crossing during seeding
+create or replace function floods.seed_legacy_crossing(
+  name text,
+  human_address text,
+  community_id integer,
+  longitude decimal,
+  latitude decimal,
+  description text default '',
+  legacy_id integer default null,
```

Comment:
maybe we should ditch legacy ids? Now that we know we can't trust them and are using lat/lng to do our legacy sync it might be worth it to just drop them entirely. (probably out of the scope of this PR but maybe worth making an issue for)